parser/src/genquery2_sql.cpp

```diff
@@ -70,9 +82,9 @@ namespace
 
 #ifdef IRODS_ENABLE_430_COMPATIBILITY
 	using log_gq             = irods::experimental::log::logger<irods::experimental::genquery2>;
-#else
+#elif !defined(IRODS_ENABLE_42X_COMPATIBILITY)
 	using log_gq             = irods::experimental::log::logger<irods::experimental::log::genquery2>;
-#endif // IRODS_ENABLE_430_COMPATIBILITY
+#endif // IRODS_ENABLE_4XX_COMPATIBILITY
```

Comment:
Same here regarding `IRODS_ENABLE_4XX_COMPATIBILITY` not being defined (or so I think)